.commitlintrc.yml

```diff
@@ -7,7 +11,7 @@ rules:
   description-empty:
     level: error
 
-  # Description must start with a capital letter and must not end with a period or space
+  # The commit description must not end with trailing white space
```

Comment:
seriously why does this matter? We have had so many issues with commit descriptions blocking releases for minor formatting issues.

Reply:
See #1066.